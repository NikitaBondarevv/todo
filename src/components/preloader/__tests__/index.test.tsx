import { render } from '@testing-library/react'

import { Preloader } from '..'

describe('<Preloader />', () => {
  test('should render <div />', () => {
    const { asFragment } = render(<Preloader />)

    expect(asFragment()).toMatchSnapshot()
  })
})
