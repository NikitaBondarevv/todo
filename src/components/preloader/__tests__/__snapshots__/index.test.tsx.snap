// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preloader /> should render <div /> 1`] = `
<DocumentFragment>
  <div
    class="preloader "
    style="width: 120px; height: 120px; border-width: 20px;"
  />
</DocumentFragment>
`;
